### DISCUSSION TITLE
Spacemacs integration

### DISCUSSION THREAD

**Discussion by bpecsek** [joaotavora/sly#412] [General]
How do I integrate sly into spacemacs to start automatically at startup or when I open a lisp file?

Now when I install sly with package-install it just vanishes for the next start of spacemacs and I have to reinstall it.

**Comment by joaotavora** [comment]
Thanks for starting this topic.  It is indeed better to start here than start an issue.  This doesn't look immediately like a SLY problem, maybe you can ping a spacemacs maintainer or at least someone knowledgeable in spacemacs.  Unfortunately, I am not that person. I think Doom Emacs has this thing called "layers" and the Common Lisp layer has SLY by default.

**Reply by bpecsek** [reply]
Thanks! I will do that.

**Reply by antoszka** [reply]
There's this: https://github.com/programingship/common-lisp-sly

It does seem to load and work more or less, but there's quite a few warnings/errors on the way, likely because it doesn't seem to have been updated for the past 7 years.

**Reply by antoszka** [reply]
Actually, scratch that, seems there's a newer fork that works without issues: https://github.com/BenedictHW/common-lisp-sly
